app/services/mentions/create_all.rb

```diff
@@ -56,7 +56,14 @@ def delete_mentions_removed_from_notifiable_text(users)
       Notification.remove_all(notifiable_ids: mentions.pluck(:id), notifiable_type: "Mention") if mentions.present?
     end
 
+    def user_has_comment_notifications?(user)
+      Notification.where(user_id: user.id,
+                         notifiable: @notifiable.id).count.positive?
```

Comment:
I think `.exists?` is faster this is case:

`user.notifications.exists?(notifiable_id: @notifiable.id)`

Reply:
Performance-wise, they are the same, but `exists?` is more succinct. Thanks.

I'd suggest `user.has_notifications_for?(@notifiable)` for readability, what do you think?

Reply:
Depends if we need a new method for multiple places, otherwise I think the one liner is fine :D